### PR TITLE
5 connect picture galleries for upcoming trip

### PR DESCRIPTION
**Pull request by ttobollik**

### CODE REVIEW

File: src/app/expedition/expedition.component.html [hunk 1]
```diff
@@ -2,71 +2,67 @@
...
         <h2>Gold</h2>
-        <h3>{{ expedition.fields.price_gold }}</h3>
+        <h2>{{ expedition.fields.price_gold }} USD*</h2>
         <p>
           {{ expedition.fields.gold_description }}
         </p>
       </div>
     </span>
+
+    <div>
```
Comment (ttobollik) [inline src/app/expedition/expedition.component.html:46]: **note:** For now hard coded. Not best practice.

File: src/app/faq/faq.component.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.description {
```
Comment (ttobollik) [inline src/app/faq/faq.component.scss:1]: **note:** Not good practice to have repeated classes in scss files. Better to set global styles and use ng styles

**Review by rochagbriel — APPROVED** [review]
All looks good for me. 👏

**Review by rochagbriel — COMMENTED** [review]
👍🏼

**Pull request merged**